help/en/html/setup/git.shtml

```diff
@@ -181,6 +165,28 @@
         <li>Preferences &#8209;&gt; Locations &#8209;&gt; User
         Files Location and set it to the JMRI folder you created in
         Dropbox.</li>
+        
+        <h2>Sneaker Net</h2>
+        
+        First, copy the <u>entire</u> directory from one of your 
+        computers to the USB stick.  This will be a master that you're moving
+        around.  (You can also directly `git clone` onto the USB stick.)
```

Comment:
A question:
I always have "Show hidden files" on my Windows computers so I see the .git folder. But I suppose normal users won't. Maybe it would be good to add a comment about that specific folder. Something like "The .git folder is usually hidden. Ensure that you copy also the hidden folders in the folder".